### PR TITLE
Add DBus method that finds DASDs for formatting

### PR DESCRIPTION
**Pull request by poncovka**
Call the DBus method `FindFormattable` of the DASD module to find
all DASDs for formatting.

### CODE REVIEW

File: pyanaconda/modules/storage/dasd/dasd.py [hunk 1]
```diff
@@ -21,20 +21,72 @@
...
+
     def publish(self):
         """Publish the module."""
         DBus.publish_object(DASD.object_path, DASDInterface(self))
 
+    @property
+    def storage(self):
+        """The storage model.
+
+        :return: an instance of Blivet
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/dasd/dasd.py]: Please document also the exception here.

Comment (poncovka) [reply pyanaconda/modules/storage/dasd/dasd.py]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Comment by poncovka** [comment]
Jenkins, test this please.

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Pull request merged**